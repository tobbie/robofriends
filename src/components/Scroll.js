import React from 'react'

const Scroll = (prop) => {
    return(
        <div style={{overflow: 'scroll', height:'800px', border:'3px solid black '}}>
            {prop.children}
        </div>
    )
}

export default Scroll