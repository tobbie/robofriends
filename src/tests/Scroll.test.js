import {shallow} from 'enzyme'
import React from 'react'
import Scroll from '../components/Scroll'

describe('scroll tests',()=> {
    
    
    it('expect to render scroll component',() => {
        expect(shallow(<Scroll/>)).toMatchSnapshot();
        
    });
})