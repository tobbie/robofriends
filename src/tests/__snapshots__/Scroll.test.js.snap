// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`scroll tests expect to render scroll component 1`] = `ShallowWrapper {}`;
